### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN018.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#64] [General]
# https://blog.3place.cn/3place/lmyn/LMYN018.html

https://blog.3place.cn/3place/lmyn/LMYN018.html

**Comment by picklecai** [comment]
工作完成了。各回各家，各找各妈。

**Comment by picklecai** [comment]
离开他连觉都睡不着。
不吃拿给他的精美食品，
如果没有他的哥哥。

哥哥不在，吃饭不香，睡觉不香。这是谁？

**Comment by picklecai** [comment]
马祠的大祭完毕以后，那些高贵尊严的神仙，每人都得到自己的一份，各自回家，纷纷四散。

公祭不宿肉在这里太有意思了。这是很明确的表达，没有一点点歧义。

**Comment by picklecai** [comment]
生了这光辉无量的儿子，㤭萨厘雅更增添了美丽，好像那生了金刚手的/群神的母亲阿底提。

歌颂母性美。

**Comment by picklecai** [comment]
光辉的大牟尼众友，
就来到了他的身旁。 （1.17.23）

他想见到那个国王，
就对他的守门人说道：
“我是迦亭〔192〕之子㤭尸迦〔193〕，
请通报国王，我已来到。” （1.17.24）

众友为什么要化名？化了个世俗意义上的名人吗？

**Reply by picklecai** [reply]
他们走进了王宫，
向甘蔗王后裔报告，
说是大仙人众友，
现在已经来到。 （1.17.26）

这也不是化名。

**Comment by picklecai** [comment]
这一个牟尼的魁首，
又问婆私吒的健康，
并对所有尊贵仙人，
按照礼节把话来讲。 （1.17.30）

礼的地位，也不低。

**Comment by picklecai** [comment]
“好像得到了醍醐；
好像大旱逢甘霖；
好像老婆生了儿子，
给没有儿子的人；
好像找到了失物；
好像幸福又临门；
您的来临就是这样，
伟大牟尼！欢迎您。 

久旱逢甘雨,，
他乡遇故知。
洞房花烛夜，
金榜题名时。

十年久旱逢甘雨，
千里他乡遇故知。
和尚洞房花烛夜，
童生金榜题名时。

何其相像！

**Reply by picklecai** [reply]
久旱逢甘雨，一滴。
他乡遇故知，借钱。
洞房花烛夜，不举。
金榜题名时，他人。

**Comment by picklecai** [comment]
您最喜欢什么事情？
能为您干点什么，我很高兴？
亲爱的！您是最高尚的人，
虔诚者！谢天谢地您来到。
今天我的生命有了报酬，
今天我的生活更加美好。

这个欢迎词，就是：有了你，世界都变亮了。
真美好呀！

**Comment by picklecai** [comment]
您从前有着王仙的称号，
苦行的光辉把您照亮，
您已经达到梵仙的地位，
我要加倍地把您颂扬。

修仙的等级链：王仙，梵仙。

**Comment by picklecai** [comment]
国王说了这些话，
把那仙人来慰问；
这些话柔和悦耳，
深深沁入他的心。

如沐春风，余音绕梁，三月不知肉味的感觉。

**Comment by picklecai** [comment]
想做什么，你不必过多地思忖。我一定都能做到，因为您就是我的神。

刚见面时，不知道对方有什么来意，说客气话，说满了。这种话，怎么能在别人提出要求前先说呢？

精通吠陀，吠陀大概没有给这方面的指导。
